### PR TITLE
[PAG-1135] activatePaymentNotice_semantic_ok.feature

### PR DESCRIPTION
**Pull request by davideromagnolieveris**
Semantic tests for activatePaymentNoticeReq with OK outcome

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/integ-test/bdd-test/features/primitives/activatePaymentNotice_semantic_ok.feature [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  # idPsp in idempotencyKey (idempotencyKey: <idPSp>+"_"+<RANDOM STRING>) not in db  [SEM_APNR_17]
+  Scenario: Check outcome OK on non-existent psp in idempotencyKey
+    Given idempotencyKey with 70000000001_UNKNOWN890 in activatePaymentNoticeReq
+    When PSP sends activatePaymentNoticeReq to nodo-dei-pagamenti
+    Then check outcome is OK
+
+  # idPsp in idempotencyKey (idempotencyKey: <idPsp>+"_"+<RANDOM STRING>) with field ENABLED = N  [SEM_APNR_18]
+  Scenario: Check outcome OK on non-existent psp in idempotencyKey
+    Given idempotencyKey with 70000000001_NOTENABLED in activatePaymentNoticeReq
```
Comment (aomegax) [inline src/integ-test/bdd-test/features/primitives/activatePaymentNotice_semantic_ok.feature:49]: @davideromagnolieveris please check input data

Comment (davideromagnolieveris) [reply src/integ-test/bdd-test/features/primitives/activatePaymentNotice_semantic_ok.feature:49]: For SEM_APNR_17 I have used idempotencyKey = 00088877799_ _randomstring_   (the first part <idPsp> corresponds to an unknown psp)

For SEM_APNR_18 I have used idempotencyKey = 80000000001_ _randomstring_   (the first part <idPsp> corresponds to a disabled psp in SIT)   --->  I have realized here that the Scenario is wrong. It should be: "Scenario: Check outcome OK on **disabled** psp in idempotencyKey"

**Review by davideromagnolieveris — COMMENTED** [review]
@aomegax

**Review by aomegax — APPROVED** [review]
done

**Review by pasqualespica — APPROVED** [review]

**Pull request merged**